db/fixtures/paths/full_stack_rails/courses/git.rb

```diff
@@ -0,0 +1,11 @@
+#######################
```

Comment:
I'm a bit torn on whether to leave the Git badge in or not. If the goal is to make this less brittle, then most solutions won't help that as they're all going to be a little hacky. Especially since we can't use a path helper that makes sense, it's just not quacking like a duck.

On the other hand it probably is good to highlight it as a skill that's taught, and taking it out removes the nice 3x3 grid symmetry. 

I experimented with a few different ways of adding the dummy Git badge but I'm not entirely convinced with any of them. The current approach (which is a bit of an experiment) is to create a dummy course and redirect it to the existing URL, which is currently the Git section in Foundations.

This felt slightly better than trying to force a struct into the view, but not by much. 

Would love to hear thoughts on this (and possible implementation alternatives if we think it's worth keeping)

Reply:
Thanks for trying all these different approaches @ChargrilledChook 💪  

It definitely would be nice to have, but I think you're right, it doesn't seem like it's worth the extra complexity for what it gives us. Let's take it out to keep it simple.